xbmc/interfaces/json-rpc/InputOperations.cpp

```diff
@@ -84,6 +88,45 @@ JSONRPC_STATUS CInputOperations::ExecuteAction(const std::string &method, ITrans
   return SendAction(action);
 }
 
+JSONRPC_STATUS CInputOperations::Button(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  std::string button = parameterObject["button"].asString();
+  std::string keymap = parameterObject["keymap"].asString();
+
+  int keycode = 0;
+  if ( keymap.compare("KB") == 0 ) // standard keyboard map
```

Comment:
you can use `operator==()` e.g. `keymap == "KB"` for the same effect with greater code clarity

Reply:
This code was actually copied from
https://github.com/xbmc/xbmc/blob/6bc162afc45fb7054d02dedb57fc4aa986f5e7cd/xbmc/network/EventClient.cpp#L62-L88

Let me think about moving it to some suitable common place as well.